isis/src/base/apps/pds2isis/pds2isis.xml

```diff
@@ -8,54 +8,64 @@ xsi:noNamespaceSchemaLocation=
 
   <description>
     <p>
-      This program converts a PDS or ISIS2 formatted file to an ISIS3 
-      <def link="Cube">cube</def> format.  The standard PDS image is defined in 
-      the "FROM" parameter, and not the "IMAGE" parameter.  If the PDS file has 
-      a detached label file and an image file, both are required to exist in the 
-      working directory.  If the label is detached, the detached label filename 
+      This program converts a PDS or ISIS2 formatted file to an ISIS3
+      <def link="Cube">cube</def> format.  The standard PDS image is defined in
+      the "FROM" parameter, and not the "IMAGE" parameter.  If the PDS file has
+      a detached label file and an image file, both are required to exist in the
+      working directory.  If the label is detached, the detached label filename
       is defined as the input file for the "FROM" parameter.  The "IMAGE" parameter
-      does not need to be defined, unless the program fails to find the image file 
+      does not need to be defined, unless the program fails to find the image file
       specifed in the label file upon the program's execution.
-      
-      The default settings normally work for most PDS products, but occasionally 
-      the user may need to modify the default values in order to obtain an image 
+
+      The default settings normally work for most PDS products, but occasionally
+      the user may need to modify the default values in order to obtain an image
       that meets their specific requirements.
      </p>
      <p>
-      The incoming <def link="Special Pixels">special pixel</def> values in the 
-      PDS file are assigned equivalent values for special pixels in ISIS3.  The 
-      order of precedence for special pixels in ISIS from highest to lowest 
-      priority is the following:  
+      The incoming <def link="Special Pixels">special pixel</def> values in the
+      PDS file are assigned equivalent values for special pixels in ISIS3.  The
+      order of precedence for special pixels in ISIS from highest to lowest
+      priority is the following:
       <blockquote>
       <ol>
-        <li><def>NULL</def></li> 
-	<li><def>HRS</def></li> 
-	<li><def>LRS</def></li> 
+        <li><def>NULL</def></li>
+	<li><def>HRS</def></li>
+	<li><def>LRS</def></li>
 	<li><def>HIS</def></li>
 	<li><def>LIS</def></li>
       </ol>
-      </blockquote> 
-      If any incoming <def link="Digital Number">pixel value</def> falls within 
-      two different special pixel types, the <def link="Special Pixel">special 
-      pixel</def> with the higher priority is assigned.  The user may also 
+      </blockquote>
+      If any incoming <def link="Digital Number">pixel value</def> falls within
+      two different special pixel types, the <def link="Special Pixel">special
+      pixel</def> with the higher priority is assigned.  The user may also
```

Comment:
This link is broken on the documentation page (https://isis.astrogeology.usgs.gov/Application/presentation/Tabbed/pds2isis/pds2isis.html). It just takes you to the top of the glossary, instead of going down to Special Pixels. For some reason this links to 'isis.astrogeology.usgs.gov*#SpecialPixel', where as the CORRECT link is 'isis.astrogeology.usgs.gov*#SpecialPixel**_s_**'